Content.Client/Atmos/Overlays/AtmosDebugBlockOverlay.cs

```diff
@@ -96,61 +96,71 @@ protected override void Draw(in OverlayDrawArgs args)
                             }
                             else if (pass == 1)
                             {
-                                // -- Blocked Directions --
-                                void CheckAndShowBlockDir(AtmosDirection dir)
```

Comment:
github diff is being fussy, I just moved these in a tab level and added if flag checks